src/MOI_wrapper.jl

```diff
@@ -26,18 +26,29 @@ const VEC_TERM = MOI.VectorAffineTerm{Float64}
 const PtrMap = Dict{Ptr{Cvoid}, Union{VarRef, ConsRef}}
 const ConsTypeMap = Dict{Tuple{DataType, DataType}, Set{ConsRef}}
 
+#will include struct SCIPData in the future
+#free_scip will also have to be adjusted
 mutable struct Optimizer <: MOI.AbstractOptimizer
-    mscip::ManagedSCIP
+    inner::SCIPData
     reference::PtrMap
     constypes::ConsTypeMap
     binbounds::Dict{VI,BOUNDS} # only for binary variables
     params::Dict{String,Any}
     start::Dict{VI,Float64} # can be partial
-    moi_separator::Any #Union{CutCbSeparator, Nothing}
+    moi_separator # ::Union{CutCbSeparator, Nothing}
```

Comment:
the type annotation should be readded

Reply:
yes I just reinserted it, don't know why I removed it in the first place